webui/react/src/ee/SamlAuth.test.ts

```diff
@@ -0,0 +1,64 @@
+import history from 'shared/routes/history';
+
+import * as utils from './SamlAuth';
```

Comment:
nit: we do name these like this in many of out tests but why not make it specific to what's being imported. eg 'saml' instead of utils

Reply:
maybe `utils` is not the best name as it's only applicable to actual utils. Maybe `target` or `subject`? The thinking is that no matter what test you are looking at, the scope of everything you want to test is under this common phrase/name...